util/execdetails/execdetails_test.go

```diff
@@ -294,11 +294,8 @@ func TestRootRuntimeStats(t *testing.T) {
 	stmtStats.RegisterStats(pid, &RuntimeStatsWithCommit{
 		Commit: commitDetail,
 	})
-	concurrency = &RuntimeStatsWithConcurrencyInfo{}
-	concurrency.SetConcurrencyInfo(NewConcurrencyInfo("concurrent", 0))
-	stmtStats.RegisterStats(pid, concurrency)
 	stats := stmtStats.GetRootStats(1)
-	expect := "time:3s, loops:2, worker:15, concurrent:OFF, commit_txn: {prewrite:1s, get_commit_ts:1s, commit:1s, region_num:5, write_keys:3, write_byte:66, txn_retry:2}"
```

Comment:
Should keep at least 1 `concurrent` information? avoid duplicate not means delete it.

Reply:
the `worker:15` is the `concurrent` information, you can see it at L283